store/shared/migrate/mysql/ddl_gen.go

```diff
@@ -186,6 +186,7 @@ func selectCompleted(db *sql.DB) (map[string]struct{}, error) {
 
 var migrationTableCreate = `
 CREATE TABLE IF NOT EXISTS migrations (
+ id INTEGER PRIMARY KEY AUTO_INCREMENT,
  name VARCHAR(255)
```

Comment:
If a primary key is required could we just remove the unique constraint from the name column, and use a primary key constraint instead?

```diff
CREATE TABLE IF NOT EXISTS migrations (
 name VARCHAR(255)
-,UNIQUE(name)
+,PRIMARY KEY(name)
)
```

Reply:
yes. I think no need `id` if `name` is unique.